### DISCUSSION TITLE
error occuring when I try to run a new react app

### DISCUSSION THREAD

**Discussion by ynsss** [babel/babel#11867] [Q&A]
So I've been trying to create a new react app using npm(yarn too) and every time I' got this error

```
    FAILED TO COMPILE


    ./src/index.js
    Error: [BABEL] C:\Users\USERNAME\Desktop\test\src\index.js: Cannot find module 'C:\Users\USERNAME\Desktop\test\node_modules\@babel\plugin-transform-

typescript\node_modules\@babel\helper-plugin-utils\lib\index.js'. Please verify that the package.json has a valid "main" entry (While processing: "C:\\Users\

\USERNAME\\Desktop\\test\\node_modules\\babel-preset-react-app\\index.js")
```

I did't change anything, just runned 
`npm(or yarn) start` 
straight after 
`npm create-react-app APP_NAME`

Also my `package.json` isn't actually different from `package.json` in my working projects

Please help 

`

**Comment by ansh8941** [comment]
C:\Users\username\Desktop\reactApp>npm install @babel/core
C:\Users\username\Desktop\reactApp>npm install babel-loader
C:\Users\username\Desktop\reactApp>npm install @babel-preset-env
C:\Users\username\Desktop\reactApp>npm install @babel/preset-react
C:\Users\username\Desktop\reactApp>npm install html-webpack-plugin

for full explanation you can visit this url: <a href="https://howdoin.com/lessons/how-do-in-setup-react-js-project-using-npm/">
									<span class="post_icon icon-book-2"></span>How do in Setup React JS Project using NPM                                </a>

**Reply by rivajunior** [reply]
"This page could not be found"

**Comment by nicolo-ribaudo** [comment]
Can you try deleting `node_modules`, the lockfile, and reinstalling everything again?
